tellapart/aurproxy/register/aws.py

```diff
@@ -156,6 +159,15 @@ def elb(self):
       self._elb = self._make_conn(boto.ec2.elb)
     return self._elb
 
+  @property
+  def elbv2(self):
+    """
+    A boto3 ELBV2 connection.
+    """
+    if not getattr(self, '_elbv2', None):
```

Comment:
Idiomatic 🐍 would be to check `x is None` instead of `not x`. I see you're repeating the existing pattern here though, so :+1:.

Reply:
yeah, there are quite a few particular style things with this codebase, good to mention though.